cypress/integration/helpMenu/HelpMenu-test-spec.js

```diff
@@ -13,15 +13,14 @@ describe('UserInfo features', () => {
     Login.relogin();
   });
 
-  it('can download the documentation PDF', () => {
+  it('can go to documentation page', () => {
     AppBar.openHelpMenu();
-    // Check that the link to the documentation PDF file exists
-    AppBar.getDocumentationLink().should('have.attr', 'target', '_blank');
+    AppBar.getDocumentationLink().find('a').should('have.attr', 'target', '_blank');
```

Comment:
Could you move the `find('a')` in the `getDocumentationLink` function please ? (idem for `getSupportPageLink`)